server/models/Document.ts

```diff
@@ -569,7 +569,7 @@ class Document extends ParanoidModel {
     ]);
 
     // Final query to get associated document data
-    const documents = await this.findAll({
+    const documents = await this.scope(["withoutState", "withDrafts"]).findAll({
```

Comment:
🙏  on including `withoutState` here though, should be a noticeable speed improvement

Reply:
Looks like [state is already excluded](https://github.com/outline/outline/blob/55e622e22ff741ad23cc52ece9b93cc9fc5bd15f/server/models/Document.ts#L83) via default scope. Will revert this change.